corehq/warehouse/transforms/sql/location_dim.sql

```diff
@@ -58,11 +58,19 @@ SELECT
     '{{ batch_id }}'
 FROM
     {{ location_staging }}
+    WHERE batch_id = {{ batch_id }}
```

Comment:
seemed like a good idea to only update what's changed in this batch rather than the entire table

Reply:
ah - I guess since the staging table get's cleared this isn't really relevant